integration-tests/support/pageObjects/createApplication-po.ts

```diff
@@ -31,3 +31,11 @@ export const componentsListPagePO = {
   applicationName: 'pf-c-title pf-m-4xl hacDev-page__heading',
   items: '[pf-c-data-list__item]',
 };
+
+export const buildApplicationWithGitSourcePagePO = {
```

Comment:
Maybe worth merging with *addComponentPagePO* as it already contains some elements of this page.

Reply:
Thanks!